apps/tenant-management-webapp-e2e/src/integration/events.feature

```diff
@@ -102,4 +102,31 @@ Feature: Events
     And the user selects "Streams" tab for "Event"
     Then the user views Core streams section
     When the user clicks eye icon of "PDF generation updates" under Core streams
-    Then the user views the details of "PDF generation updates" under Core streams
\ No newline at end of file
+    Then the user views the details of "PDF generation updates" under Core streams
+
+  @TEST_CS-1480 @REQ_CS-1352 @regression
+  Scenario: As a tenant admin, I can add, edit and delete an event stream
+    Given a tenant admin user is on event streams page
+    # Add a stream
+    When the user clicks Add stream button
+    Then the user views Add stream modal
+    When the user enters "autotest-addEditDelete-stream", "autotest-stream-desc", "status-service:application-healthy, file-service:file-deleted", "public" in stream modal
+    And the user clicks Save button in Stream modal
+    Then the user "views" the stream of "autotest-addEditDelete-stream", "Public"
+    When the user clicks "Eye" button of "autotest-addEditDelete-stream"
+    Then the user views the stream details of "autotest-addEditDelete-stream", "autotest-stream-desc", "status-service:application-healthy, file-service:file-deleted", "public"
+    And the user clicks "Eye-Off" button of "autotest-addEditDelete-stream"
+    # Edit a stream
+    When the user clicks "Edit" button of "autotest-addEditDelete-stream"
+    Then the user views Edit stream modal
+    When the user removes event chips of "status-service:application-healthy" in stream modal
+    And the user enters "autotest-stream-desc2", "auto-test-role3, beta-tester" in stream modal
```

Comment:
When the user enters "autotest-stream-desc2", "EMPTY", "auto-test-role3, beta-tester" in Edit stream modal

Reply:
Added Edit to the name of the test case and "n/a"